### PR TITLE
chore: change locators for splash screens and divide them into 2

### PR DESCRIPTION
**Pull request by anastasiyaig**
Apparently, we have 2 splash screens:

1. splash screen with status spinning icon
2. splash screen with loading animation of words under Did you know section

They are being displayed one by one, so i decided it would be helpful to clarify which screen we are waiting for. It is a cosmetic change, no extra functionality added.

### CODE REVIEW

**Comment by anastasiyaig** [comment]
There is an ongoing process to disable second screen (DidYouKnow) for builds when we have env variable TEST MODE = true , u can check it in https://github.com/status-im/status-desktop/issues/12736. When it is implemented it will reduce time of tests for 30s per each app launch

File: tests/onboarding/test_onboarding_import_seed.py [hunk 1]
```diff
@@ -7,7 +9,8 @@
 from driver.aut import AUT
 from gui.components.onboarding.before_started_popup import BeforeStartedPopUp
 from gui.components.onboarding.beta_consent_popup import BetaConsentPopup
-from gui.components.splash_screen import SplashScreen
+from gui.components.splash_screen_did_u_know import SplashScreenDidYouKnow
```
Comment (Valentina1133) [inline tests/onboarding/test_onboarding_import_seed.py:12]: No need to import SplashScreenDidYouKnow ? Don't see it in test

File: tests/onboarding/test_onboarding_generate_new_keys.py [hunk 2]
```diff
@@ -8,7 +8,8 @@
 from gui.components.onboarding.before_started_popup import BeforeStartedPopUp
 from gui.components.onboarding.beta_consent_popup import BetaConsentPopup
 from gui.components.picture_edit_popup import shift_image
-from gui.components.splash_screen import SplashScreen
+from gui.components.splash_screen_did_u_know import SplashScreenDidYouKnow
```
Comment (Valentina1133) [inline tests/onboarding/test_onboarding_generate_new_keys.py:11]: Here also
No need to import SplashScreenDidYouKnow ? Don't see it in test

File: gui/screens/onboarding.py [hunk 3]
```diff
@@ -12,7 +12,8 @@
 from driver.objects_access import walk_children
 from gui.components.os.open_file_dialogs import OpenFileDialog
 from gui.components.picture_edit_popup import PictureEditPopup
-from gui.components.splash_screen import SplashScreen
+from gui.components.splash_screen_did_u_know import SplashScreenDidYouKnow
+from gui.components.splash_screen_main_loader import SplashScreenMainLoader
```
Comment (Valentina1133) [inline gui/screens/onboarding.py:16]: Same
No need to import SplashScreenDidYouKnow ? Don't see it in test

File: gui/main_window.py [hunk 4]
```diff
@@ -10,7 +10,8 @@
 from gui.components.community.invite_contacts import InviteContactsPopup
 from gui.components.onboarding.before_started_popup import BeforeStartedPopUp
 from gui.components.onboarding.beta_consent_popup import BetaConsentPopup
-from gui.components.splash_screen import SplashScreen
+from gui.components.splash_screen_did_u_know import SplashScreenDidYouKnow
```
Comment (Valentina1133) [inline gui/main_window.py:13]: Same 
No need to import SplashScreenDidYouKnow ? Don't see it in test

**Review by Valentina1133 — APPROVED** [review]

File: tests/onboarding/test_onboarding_import_seed.py (hunk 1 above)
Comment (anastasiyaig) [reply tests/onboarding/test_onboarding_import_seed.py]: yes, sure, i forgot, sorry about that

**Review by StateOf-Vlado — APPROVED** [review]

**Pull request closed without merging**